src/components/ValidatedInput/ValidSelectAsync.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
export this component in /components/index?

Reply:
updated